src/Hosting/Systemd/test/Microsoft.Extensions.Hosting.Systemd.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <Reference Include="Microsoft.Extensions.Hosting.Systemd" />
     <Reference Include="Microsoft.Extensions.Hosting" />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <Reference Include="Microsoft.Extensions.Logging.Console" />
```

Comment:
This was added to add a reference so on test failure a more reasonable assert failure will occur instead of a cryptic assembly load error.